web/static/as_printable_html/book.css

```diff
@@ -152,16 +174,16 @@
     margin: 0;
     padding: 0;
   }
-
-  section.depth-2.legaldocument {
-    break-before: page;
+  .highlighted {
+    background: var(--highlight-background-color);
+    padding: calc(var(--casebook-line-height) * .2) 0;
   }
-
-  sup {
-    font-size: calc(var(--casebook-font-size) / 2.5);
-
+  /* Support for greyscale-only printing */
+  @media not color {
+    .highlighted {
+      background: var(--highlight-background-greyscale);
+    }
```

Comment:
I can't find any indication of whether this is supported anywhere (Chrome does not respect it when I print-preview on a b/w printer—I get its own greyscale rendition of the highlight color). But the spec says this should work as intended.